joomla/joomla-cms
# [bug] Support translated layout names

Pull Request for Issue #39306 [partially].

### Summary of Changes

If a module layout uses a language string, then a child template override should also correctly use that.

### Testing Instructions

- Create a child template of cassiopeia.
- copy the module menu overrides to the child
- check the module layout select element


### Actual result BEFORE applying this Pull Request

![Screenshot 2022-11-25 at 20 09 17](https://user-images.githubusercontent.com/3889375/204043761-2c09144c-33f8-4a9b-9a8a-fac19511ca20.png)


### Expected result AFTER applying this Pull Request

![Screenshot 2022-11-25 at 20 09 48](https://user-images.githubusercontent.com/3889375/204043784-bdb89cb9-bc3d-4ab6-8e56-6c79624d133f.png)


### Link to documentations
Please select:
